web/stats.php

```diff
@@ -404,9 +414,14 @@ function by_sent($a, $b) {
     # Output data
     global $year_bar;
     global $navigation_tabs;
+    if ($year == '2015') {
+        $title_string = "WriteToThem.com Zeitgeist 2015&ndash;2016";
+    } else {
+        $title_string = "WriteToThem.com Zeitgeist $year";
+    }
     template_draw($xml ? 'stats-mp-twfy' : 'stats-mp-performance', array(
-        "title" => "WriteToThem.com Zeitgeist $year",
-        'year' => $year,
+        "title" => $title_string,
+        'year' => $year == '2015' ? '2015&ndash;16' : $year,
```

Comment:
This is breaking the sharing URLs which use $year when they are being constructed.